src/components/Gradebook/index.jsx

```diff
@@ -285,21 +285,23 @@ export default class Gradebook extends React.Component {
 
   handleClickExportGrades = () => {
     trackEvent(() => ({
-      name: 'edx.gradebook.reports.grade_export',
+      name: 'edx.gradebook.reports.grade_export.downloaded',
       properties: {
         category: trackingCategory,
         courseId: this.props.courseId,
+        label: this.props.courseId,
```

Comment:
What are we "checking"? By that do you mean that the need for this line of code may be temporary? If so, under what circumstances can we be sure it can be removed?

Reply:
Google analytics has the following types: 
event category ( gradebook - currently used)
name: (which we use for event action)
label - currently not used
value - currently not used

I want to check that by setting label, we will indeed store the courseId in the label of the event. If it works, we may want to add label to all events, so we could at least track courseIds in google analytics